content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -9,44 +9,99 @@ weight: 210
 ---
 
 <!-- overview -->
-This page shows how to enable and configure encryption of secret data at rest.
+
+Some of the APIs in Kubernetes, such as {{< glossary_tooltip text="Secret" term_id="secret" >}},
+support at-rest encryption. This at-rest encryption is additional to any system-level
```

Comment:
I'm not sure if the encryption is supported on a per-API basis. The encryption configuration actually
allows us to choose which GVK to encrypt. It is okay to encrypt Deployments instead of Secrets.

```suggestion
Kubernetes support at-rest encryption for resources. The encryption is useful for some sensitive data
such as {{< glossary_tooltip text="Secret" term_id="secret" >}} objects.
This at-rest encryption is additional to any system-level
```

Reply:
You can't choose which resources are and aren't encrypted, not in the HTTP sense of resource (each resource in HTTP has a unique URI). You do indeed configure it at the group / kind level, which is what I mean by ”per API”.

Reply:
Emm ... I'd disagree with you on the use of 'API' here. You can say you are encrypting an API resource, an API kind.
In this context, we are not talking about HTTP resources. K8s has a narrow definition of resources.